### ISSUE TITLE
Is it possible to add images dynamicaly into a text frame?

### ISSUE THREAD

**Issue by flagadur** [berteh/ScribusGenerator#213]
It would be super awesome to insert images into text frame dynamically. I don't think it's possible yet, is it? Thanks ;)

Example : 
My Text_Frame : "%VAR_my_text%
my_text : "Inflict 3 %VAR_image_for_damage% to the target %VAR_image_for_creature%

**Comment by berteh** [comment]
It is perfectly feasible... and was actually one of my reasons to develop the tool, just not *in* a single dynamic text but *between* multiple dynamic texts (that can all be inserted in a single text frame, in sequence). 

Since variables (text or images) that would be empty after the substitution are removed by ScribusGenerator you should be able to get the expected result.

To take your example I'd cut the data somewhere along the table below, and design the Scribus template accordingly while playing with the anchor of the image frames so that it keeps aligned with the preceding character. Empty texts and images placeholders will be remove when empty, no worry there.

| text1 | img12 | text2 | img23 | text3 | img34 | text4 |
|--------|--------|--------|--------|--------|--------|--------|
| Inflict 3 | ./pic/dmg-inline.png | to the target | ./pic/creature/basilik-inline.png |  |  |  |
| Inflict 1 | ./pic/dmg-inline.png | to every | ./pic/creature/blue.png | creature in the oponent field. |  |  |
| Recover 3 | ./pic/health-inline.png | and add 2 | ./pic/defence-inline.png | tokens to all your |  ./pic/creature/purple.png | creatures |

**Comment by berteh** [comment]
you're right. the "inline" object didn't exist yet so I played with absolute positionning and text alignement (right align for text left of the image, for instance) to mimic some inline.

It should be possible to adapt the SG code to handle inline objects without too much work, will have a look but I don't exactly know when... if you wanna give it a shot feel free to hack the code and submit a pull request.
